config/config.go

```diff
@@ -378,6 +379,7 @@ func NewConfig() *Config {
 	flag.IntVar(&cfg.BlockProfileRate, "block-profile-rate", 0, "block profile sample rate, see runtime.SetBlockProfileRate")
 	flag.IntVar(&cfg.MutexProfileFraction, "mutex-profile-fraction", 0, "mutex profile fraction rate, see runtime.SetMutexProfileFraction")
 	flag.IntVar(&cfg.MemProfileRate, "memory-profile-rate", 0, "memory profile rate, see runtime.SetMemProfileRate, keeps default 512 kB")
+	flag.BoolVar(&cfg.EnableStartTimestamp, "enable-start-timestamp", false, "only for prometheus: adds start label to each counter with the value of counter creation timestamp as unix nanoseconds")
```

Comment:
Maybe better to have also `label` in this name and flag

Reply:
Fixed, thanks